.env.example

```diff
@@ -5,3 +5,28 @@ USER_LIST=user-name1, user-name2
 
 DURATION_IN_DAYS=5
 PAGE_DATA_COUNT=5
+
+# score weights for each of the sub types. For more details on the sub-types, please refer to README.md
+
+# score weights for various sub-type of 'PR opened' type
+T1S1=1
+T1S2=1
+
+# score weights for various sub-type of 'PR opened' type
+T2S1=0.5
+T2S2=0.5
+T2S3=0.5
+T2S4=0.5
+
+# score weights for various sub-type of 'Issue created' type
+T3S1=0.3
+T3S2=0.3
+
+# score weights for various sub-type of 'PR/Issue comments' type
+T4S1=0.1
+T4S2=0.1
+T4S3=0.1
+T4S4=0.1
+
+# score weight for 'Repo created' type
+T5S1=0
```

Comment:
```suggestion
T5S1=0

```

Reply:
On the second thought, we don't need these in the envrionment. These are a part of internal configurations.

Reply:
@kabirbaidhya I've reverted it back to how it was and just changes the score weights.